libpod/finished_unsupported.go

```diff
@@ -0,0 +1,12 @@
+// +build darwin,!linux
```

Comment:
Shouldn't this have been "everything not covered by `finished_32.go` and `finished_64.go`"?